### PR TITLE
Update LCF_ChangeLog.md

### PR DESCRIPTION
**Pull request by anthonywhitford**
Updated change log inline with review group meeting from 9th April. #336

### CODE REVIEW

File: docs/LCF_ChangeLog.md
Comment (franciscave) [inline docs/LCF_ChangeLog.md:1]: Requested changes in line 23: deprecated -> deprecate, added -> add. Rationale: to make consistent use of the present tense.

**Review by franciscave — CHANGES_REQUESTED** [review]

File: docs/LCF_ChangeLog.md
Comment (franciscave) [inline docs/LCF_ChangeLog.md:1]: Requested change in line 44: affected -> affect. Rationale: consistent use of present tense, as in line 23.

**Review by franciscave — CHANGES_REQUESTED** [review]

**Comment by franciscave** [comment]
Apart from the requested changes in two lines, I think this is great.

File: docs/LCF_ChangeLog.md
Comment (anthonywhitford) [reply docs/LCF_ChangeLog.md:1]: Done

Comment (anthonywhitford) [reply docs/LCF_ChangeLog.md:1]: Done

**Review by franciscave — APPROVED** [review]
Thanks for making the corrections. I think that, for consistency, there may need to be a blank line added between lines 32 and 33, i.e. above the heading 'Payment modifications'. Other than that, it looks good.

**Comment by anthonywhitford** [comment]
@franciscave I've added in the blank line on lines 32/33 as requested.

**Comment by anthonywhitford** [comment]
@mdovey Could you review and (hopefully) approve please? The branch rules require two approvers before a merge is possible.

**Review by mdovey — APPROVED** [review]

**Pull request merged**
